gse_account/models/company.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models, api
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    use_custom_msg = fields.Boolean('Custom Invoice Messages', config_parameter='account.use_custom_msg')
+    company_specific = fields.Html(string='Company Specific', translate=True)
```

Comment:
@MaganaAsiati @iambenkis 
'company_specifc' seems strange as field name.
I'd rather call it 'custom_invoice_message'
and the string Singular 'Custom Invoice Message'
What do you think?